### ISSUE TITLE
Submenus not navigable via keyboard

### ISSUE THREAD

**Issue by Nealium104** [uklibraries/UKL_HeaderFooter#60]
Users should be able to navigate all links within the header via keyboard for accessibility reasons. Currently, items within the dropdown menu are not navigable by keyboard (via "tab").

**Comment by Nealium104** [comment]
Menus navigable as of commit #a3e3b48

**Comment by Nealium104** [comment]
Fixed with a3e3b48, more work to be done on the click behavior that will be moved to branch click-behavior
